src/main/java/models/games/abstracts/AbstractGame.java

```diff
@@ -0,0 +1,43 @@
+package models.games.abstracts;
+
+import interfaces.Content;
+import interfaces.Games;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+import models.GameTypes;
+import models.User;
+
+import java.util.Objects;
+@Data
+public abstract class AbstractGame implements Games {
```

Comment:
Why do we need @Setter and @Getter annotation if we have @Data annotation?
Why not use lombok to generate equals & hashCode and toString?

Reply:
- [x] Used only @Data annotation and added @ToString and @EqualsAndHashCode annotations into AbstractGame and User classes